cdap-ui/app/cdap/components/PipelineSummary/index.js

```diff
@@ -130,7 +130,11 @@ export default class PipelineSummary extends Component {
     }
     Mousetrap.unbind('esc');
   }
-  componentDidMount() {
+
+  fetchStats = () => {
+    if (isBatchPipeline(this.props.pipelineType)) {
```

Comment:
shouldn't this be `if (!...)`? We do want to fetch stats if it is batch pipeline right?